### PR TITLE
[hi] Add content/hi/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro.html

### PR DESCRIPTION
**Pull request by rajatgupta24**
Hey folks, this is a PR for hi localization of the file: `content/hi/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro.html`
PR is linked under this issue #29353

/language hi

### CODE REVIEW

File: content/hi/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro.html [hunk 1]
```diff
@@ -0,0 +1,106 @@
+---
+title: डिप्लॉयमेंट बनाने के लिए कुबेक्टल का उपयोग करना
+weight: 10
+---
+
+<!DOCTYPE html>
+
+<html lang="en">
```
Comment (sftim) [inline content/hi/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro.html:8]:
```suggestion
<html lang="hi">
```

Comment (rajatgupta24) [reply content/hi/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro.html]: Thanks, @sftim, I made the change.

**Comment by anubha-v-ardhan** [comment]
/retitle [hi] Add content/hi/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro.html

File: content/hi/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro.html [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+
+    <main class="content">
+
+        <div class="row">
+
+         <div class="col-md-8">
+          <h3>उद्देश्यों</h3>
+                <ul>
+                    <li>एप्लिकेशन डिप्लॉयमेंट के बारे में जानें।</li>
+                    <li>कुबेरनेट्स पर कुबेक्टल के साथ अपना पहला ऐप डिप्लॉय करें।</li>
```
Comment (anubha-v-ardhan) [inline content/hi/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro.html:24]:
```suggestion
                    <li>कुबेरनेट्स पर kubectl के साथ अपना पहला ऐप डिप्लॉय करें।</li>
```

File: content/hi/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro.html [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+                </ul>
+            </div>
+
+            <div class="col-md-8">
+                <h3>कुबेरनेट्स डिप्लॉयमेंट</h3>
+
+                <p>एक बार जब आपके पास कुबेरनेट्स क्लस्टर चल रहा हो, तो आप इसके ऊपर अपने कंटेनरीकृत अनुप्रयोगों को तैनात कर सकते हैं। ऐसा करने के लिए, आप कुबेरनेट्स <b>डिप्लॉयमेंट</b> कॉन्फ़िगरेशन बनाते हैं। डिप्लॉयमेंट कुबेरनेट्स को निर्देश देता है कि आपके आवेदन के उदाहरण कैसे बनाएं और अपडेट करें। एक बार जब आप एक डिप्लॉयमेंट बना लेते हैं, तो कुबेरनेट्स कंट्रोल प्लेन उस डिप्लॉयमेंट में शामिल एप्लिकेशन इंस्टेंस को क्लस्टर में अलग-अलग नोड्स पर चलाने के लिए शेड्यूल करता है।</p>
```
Comment (anubha-v-ardhan) [inline content/hi/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro.html:31]:
```suggestion
                <p>एक बार जब आपके पास कुबेरनेट्स क्लस्टर चल रहा हो, तो आप इसके ऊपर अपने कंटेनरीकृत एप्लीकेशन को तैनात कर सकते हैं। ऐसा करने के लिए, आप कुबेरनेट्स <b>डिप्लॉयमेंट</b> कॉन्फ़िगरेशन बनाते हैं। डिप्लॉयमेंट कुबेरनेट्स को निर्देश देता है कि आपके आवेदन के उदाहरण कैसे बनाएं और अपडेट करें। एक बार जब आप एक डिप्लॉयमेंट बना लेते हैं, तो कुबेरनेट्स कंट्रोल प्लेन उस डिप्लॉयमेंट में शामिल एप्लिकेशन इंस्टेंस को क्लस्टर में अलग-अलग नोड्स पर चलाने के लिए शेड्यूल करता है।</p>
```

File: content/hi/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro.html [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+
+                <p>एक बार एप्लिकेशन इंस्टेंस बन जाने के बाद, कुबेरनेट्स डिप्लॉयमेंट कंट्रोलर लगातार उन इंस्टेंस की निगरानी करता है। यदि किसी इंस्टेंस को होस्ट करने वाला नोड नीचे चला जाता है या हटा दिया जाता है, तो डिप्लॉयमेंट कंट्रोलर क्लस्टर में किसी अन्य नोड पर इंस्टेंस को इंस्टेंस से बदल देता है। <b>यह मशीन की विफलता या रखरखाव को दूर करने के लिए एक स्व-उपचार तंत्र प्रदान करता है।</b></p>
```
Comment (anubha-v-ardhan) [inline content/hi/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro.html:33]:
```suggestion
                <p>एक बार एप्लिकेशन इंस्टेंस बन जाने के बाद, कुबेरनेट्स डिप्लॉयमेंट कंट्रोलर लगातार उन इंस्टेंस की निगरानी करता है। यदि किसी इंस्टेंस को होस्ट करने वाला नोड बंद हो जाता है या हटा दिया जाता है, तो डिप्लॉयमेंट कंट्रोलर क्लस्टर में इंस्टेंस को किसी अन्य नोड के इंस्टेंस से बदल देता है। <b>यह मशीन की विफलता या रखरखाव को दूर करने के लिए एक स्व-उपचार तंत्र प्रदान करता है।</b></p>
```

File: content/hi/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro.html [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+
+                <p>पूर्व-ऑर्केस्ट्रेशन की दुनिया में, इंस्टॉलेशन स्क्रिप्ट का उपयोग अक्सर अनुप्रयोगों को शुरू करने के लिए किया जाता था, लेकिन वे मशीन की विफलता से पुनर्प्राप्ति की अनुमति नहीं देते थे। कुबेरनेट्स डिप्लॉयमेंट आपके एप्लिकेशन इंस्टेंस को बनाकर और उन्हें नोड्स में चालू रखते हुए, एप्लिकेशन प्रबंधन के लिए एक मौलिक रूप से अलग दृष्टिकोण प्रदान करता है। </p>
```
Comment (anubha-v-ardhan) [inline content/hi/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro.html:35]:
```suggestion
                <p>पूर्व-ऑर्केस्ट्रेशन की दुनिया में, इंस्टॉलेशन स्क्रिप्ट का उपयोग अक्सर एप्लीकेशन को शुरू करने के लिए किया जाता था, लेकिन वे मशीन की विफलता से पुनर्प्राप्ति की अनुमति नहीं देते हैं। कुबेरनेट्स डिप्लॉयमेंट आपके एप्लिकेशन इंस्टेंस को बनाकर और उन्हें नोड्स पर चालू रखते हुए, एप्लिकेशन प्रबंधन के लिए एक मौलिक रूप से अलग दृष्टिकोण प्रदान करता है। </p>
```

File: content/hi/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro.html [hunk 6]
```diff
@@ -0,0 +1,106 @@
...
+
+            </div>
+
+            <div class="col-md-4">
+                <div class="content__box content__box_lined">
+                    <h3>सारांश:</h3>
+                    <ul>
+                        <li>डिप्लॉयमेंट</li>
+                        <li>कुबेक्टल</li>
```
Comment (anubha-v-ardhan) [inline content/hi/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro.html:44]:
```suggestion
                        <li>kubectl</li>
```

File: content/hi/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro.html [hunk 7]
```diff
@@ -0,0 +1,106 @@
...
+        <div class="row">
+            <div class="col-md-8">
+                <p><img src="/docs/tutorials/kubernetes-basics/public/images/module_02_first_app.svg"></p>
+            </div>
+        </div>
+        <br>
+
+        <div class="row">
+            <p class="col-md-8">
+                <p>आप कुबेरनेट्स कमांड लाइन इंटरफेस, <b>कुबेक्टल</b> का उपयोग करके डिप्लॉयमेंट बना और प्रबंधित कर सकते हैं। कुबेक्टल क्लस्टर के साथ बातचीत करने के लिए कुबेरनेट्स एपीआई का उपयोग करता है। इस मॉड्यूल में, आप कुबेरनेट्स क्लस्टर पर आपके एप्लिकेशन चलाने वाले डिप्लॉयमेंट बनाने के लिए आवश्यक सबसे सामान्य कुबेक्टल कमांड सीखेंगे।</p>
```
Comment (anubha-v-ardhan) [inline content/hi/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro.html:69]:
```suggestion
                <p>आप कुबेरनेट्स कमांड लाइन इंटरफेस, <b>kubectl</b> का उपयोग करके डिप्लॉयमेंट बना और प्रबंधित कर सकते हैं। kubectl क्लस्टर के साथ बातचीत करने के लिए कुबेरनेट्स एपीआई का उपयोग करता है। इस मॉड्यूल में, आप कुबेरनेट्स क्लस्टर पर आपके एप्लिकेशन चलाने वाले डिप्लॉयमेंट बनाने के लिए आवश्यक सबसे सामान्य kubectl कमांड सीखेंगे।</p>
```

File: content/hi/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro.html [hunk 8]
```diff
@@ -0,0 +1,106 @@
...
+
+                <p>जब आप कोई डिप्लॉयमेंट बनाते हैं, तो आपको अपने एप्लिकेशन के लिए कंटेनर इमेज और चलाने के लिए इच्छित प्रतिकृतियों की संख्या निर्दिष्ट करने की आवश्यकता होगी। आप अपने कामकाज को अपडेट करके बाद में उस जानकारी को बदल सकते हैं; मॉड्यूल <a href="/hi//docs/tutorials/kubernetes-basics/scale/scale-intro/">5</a> और <a href="/hi/docs/tutorials/kubernetes-basics/update/update-intro/" >6</a> बूटकैंप के बारे में चर्चा करें कि आप अपने डिप्लॉयमेंट को कैसे स्केल और अपडेट कर सकते हैं।</p>
```
Comment (anubha-v-ardhan) [inline content/hi/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro.html:71]:
```suggestion
                <p>जब आप कोई डिप्लॉयमेंट बनाते हैं, तो आपको अपने एप्लिकेशन के लिए कंटेनर इमेज और चलाने के लिए इच्छित प्रतिकृतियों की संख्या निर्दिष्ट करने की आवश्यकता होगी। आप अपने कामकाज को अपडेट करके बाद में उस जानकारी को बदल सकते हैं; बूटकैंप के मॉड्यूल <a href="/hi//docs/tutorials/kubernetes-basics/scale/scale-intro/">5</a> और <a href="/hi/docs/tutorials/kubernetes-basics/update/update-intro/" >6</a> चर्चा करते हैं कि आप अपने डिप्लॉयमेंट को कैसे स्केल और अपडेट कर सकते हैं।</p>
```

File: content/hi/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro.html [hunk 9]
```diff
@@ -0,0 +1,106 @@
...
+
+
+
+            </div>
+            <div class="col-md-4">
+                <div class="content__box content__box_fill">
+                    <p><i> कुबेरनेट्स पर डिप्लॉयड होने के लिए अनुप्रयोगों को समर्थित कंटेनर प्रारूपों में से एक में पैक करने की आवश्यकता है </i></p>
```
Comment (anubha-v-ardhan) [inline content/hi/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro.html:78]:
```suggestion
                    <p><i> कुबेरनेट्स पर डिप्लॉयड होने के लिए एप्लीकेशन को समर्थित कंटेनर प्रारूपों में से एक में पैक करने की आवश्यकता है </i></p>
```

File: content/hi/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro.html [hunk 10]
```diff
@@ -0,0 +1,106 @@
...
+                </div>
+            </div>
+        </div>
+
+        <div class="row">
+            <div class="col-md-8">
+                <p>
+                    अपने पहले डिप्लिव के लिए, आप एक डॉकर कंटेनर में पैक किए गए हैलो-नोड एप्लिकेशन का उपयोग करेंगे जो सभी अनुरोधों को प्रतिध्वनित करने के लिए NGINX का उपयोग करता है। (यदि आपने पहले से हैलो-नोड एप्लिकेशन बनाने और कंटेनर का उपयोग करके इसे तैनात करने का प्रयास नहीं किया है, तो आप पहले <a href="/hi/docs/tutorials/hello-minikube/">हेल्लो Minikube के निर्देशों का पालन करके ऐसा कर सकते हैं। ट्यूटोरियल </a>)।
```
Comment (anubha-v-ardhan) [inline content/hi/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro.html:86]:
```suggestion
                    अपने पहले डिप्लॉयमेंट के लिए, आप एक डॉकर कंटेनर में पैक किए गए हैलो-नोड एप्लिकेशन का उपयोग करेंगे जो सभी अनुरोधों को प्रतिध्वनित करने के लिए NGINX का उपयोग करता है। (यदि आपने पहले से हैलो-नोड एप्लिकेशन बनाने और कंटेनर का उपयोग करके इसे तैनात करने का प्रयास नहीं किया है, तो आप पहले <a href="/hi/docs/tutorials/hello-minikube/">हेलो Minikube ट्यूटोरियल</a> के निर्देशों का पालन करके ऐसा कर सकते हैं)।
```

**Review by anubha-v-ardhan — CHANGES_REQUESTED** [review]
:)

File: content/hi/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro.html [hunk 11]
```diff
@@ -0,0 +1,106 @@
+---
+title: डिप्लॉयमेंट बनाने के लिए कुबेक्टल का उपयोग करना
```
Comment (anubha-v-ardhan) [inline content/hi/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro.html:2]:
```suggestion
title: डिप्लॉयमेंट बनाने के लिए kubectl का उपयोग करना
```

**Comment by anubha-v-ardhan** [comment]
@rajatgupta24 Can you look at the suggestions above. We could merge this PR then

**Comment by anubha-v-ardhan** [comment]
@rajatgupta24 Are you active with this PR? Can you have a look at suggestions above

**Comment by anubha-v-ardhan** [comment]
/lgtm
/approve
Thank you @rajatgupta24

**Pull request merged**